shell/browser/ui/web_contents_modal_dialog_host_views.cc

```diff
@@ -0,0 +1,72 @@
+#include "components/web_modal/web_contents_modal_dialog_host.h"
+#include "shell/browser/native_window.h"
+#include "ui/gfx/geometry/point.h"
+#include "ui/gfx/geometry/rect.h"
+#include "ui/gfx/geometry/size.h"
+#include "ui/views/widget/widget.h"
+#include "ui/views/window/client_view.h"
+
+using views::View;
+using web_modal::ModalDialogHostObserver;
+using web_modal::WebContentsModalDialogHost;
+
+constexpr int kDialogTop = 64;
```

Comment:
This seems tuned for Chrome's tab + address bar. Should this be customizable by apps?

Reply:
Maybe an API like: `webContents.setModalDialogOptions({ top: 64 })`?

Reply:
Seems like it should maybe be on BrowserWindow rather than WebContents..? and should maybe default to `0`, similarly to how the file open/save dialogs appear at 0px?